coffee-model/coffee-model-base/src/main/java/hu/icellmobilsoft/coffee/model/base/IIdentifiedAuditEntity.java

```diff
@@ -0,0 +1,118 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.coffee.model.base;
+
+import java.io.Serializable;
+
+/**
+ * Interface for the identified audit entities. Contains setters, getters for id, version, and for the basic audit fields.
+ *
+ * @param <ID> The type of the entity id
+ * @param <USER> The type of the creator and modifier user
+ * @param <DATE> The type of the date. E.g.: {@link java.util.Date}, {@link java.time.OffsetDateTime}, {@link java.time.Instant}
+ * @author martin.nagy
+ * @since 1.8.0
+ */
+public interface IIdentifiedAuditEntity<ID extends Serializable, USER, DATE> {
+
+    /**
+     * Returns the id of the entity
+     *
+     * @return the id of the entity
+     */
+    ID getId();
+
+    /**
+     * Sets the id of the entity
+     *
+     * @param id the new id
+     */
+    void setId(ID id);
+
+    /**
+     * Returns the version of the entity
+     *
+     * @return the version of the entity
+     */
+    long getVersion();
```

Comment:
Ez az abstract osztályokban is külön van kezelve, lehet érdemes lenne interface szinten is kiszervezni külön